static/app/views/dashboardsV2/detail.spec.jsx

```diff
@@ -943,7 +932,7 @@ describe('Dashboards > Detail', function () {
             'dashboards-basic',
             'dashboards-edit',
             'discover-query',
-            'dashboard-grid-layout',
+
```

Comment:
```suggestion
```
nit: Can we delete some of the whitespace lol

Reply:
Damn it. I was hoping that my eslint auto-fix stuff would've gotten rid of it. Or prettier. lol
I just did a replace for `s/'dashboard-grid-layout',//`